time_weighted_average_price/on_chain/package-lock.json

```diff
@@ -1858,7 +1858,8 @@
       "version": "1.0.9",
       "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.0.9.tgz",
       "integrity": "sha512-LEyx4aLEC3x6T0UguF6YILf+ntvmOaWsVfENmIW0E9H09vKlLDGelMjjSm0jkDHALj8A8quZ/HapKNigzwge+Q==",
-      "license": "ISC"
+      "license": "ISC",
+      "peer": true
```

Comment:
What's this diff from? Is this from the new version of node?

Reply:
ya bumping to node v22 caused this update